extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -151,12 +152,44 @@ public String getThumbnailUrl() throws ParsingException {
     public String getDescription() throws ParsingException {
         assertPageFetched();
         try {
-            return doc.select("p[id=\"eow-description\"]").first().html();
+            return fixDescriptionLinks(doc.select("p[id=\"eow-description\"]").first().html());
         } catch (Exception e) {//todo: add fallback method <-- there is no ... as long as i know
             throw new ParsingException("Could not get the description", e);
         }
     }
 
+    private String fixDescriptionLinks(String description) throws ParsingException, UnsupportedEncodingException {
+        boolean everythingIsFine = true;
+        String descriptionE = java.net.URLDecoder.decode(description, "UTF-8");
+        Parser.getLinksFromString(descriptionE);
+        int endlessloop = 0;
+        while(everythingIsFine) {
+            try {
+                String[] FirstCut = description.split("<a href=\"",2);
+                if(FirstCut.length==1) {
+                    everythingIsFine = false;
+                }
+
+                String Beginning = FirstCut[0];
+                String[] SecondCut = FirstCut[1].split("\"",2);
```

Comment:
what are you going todo if ther is a `\\"` in the url which gets encoded into a `\"`